### PR TITLE
Added build-android script

### PR DESCRIPTION
**Pull request by amraboelela**

### CODE REVIEW

**Comment by johnno1962** [comment]
Hi @amraboelela, see if you can slave a few tips from https://github.com/apple/swift-corelibs-foundation/pull/1113/files which has some different takes on thread names, NI_MAXHOST etc.

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,5 @@ Build
 *.swp
 *.orig
 .arcconfig
+
+build.ninjae
```
Comment (spevans) [inline .gitignore]: Is this meant to be `build.ninja`? If so it is already covered by a wildcard

**Comment by spevans** [comment]
@amraboelela Could you rebase this onto master just to eliminate the merge commits please.

File: .gitignore (hunk 1 above)
Comment (johnno1962) [reply .gitignore]: This is probably coming from the "sed -ie” commands in the script where “e” is taken as the name for the temp file.

**Comment by alblue** [comment]
Would also help to squash the commits and improve the commit messages for future readability.

**Comment by amraboelela** [comment]
Created new PR https://github.com/apple/swift-corelibs-foundation/pull/1249 

Closing this one

**Pull request closed without merging**